test/index.js

```diff
@@ -12,6 +12,7 @@ describe('env-var', function () {
     INVALID_BASE_64: 'a|GV-sb*G8=',
     STRING: 'oh hai',
     FLOAT: '12.43',
+    FLOAT_INTEGER: '1.0',
```

Comment:
Can you add another assertion for a variable such as `INVALID_FLOAT: '1.0oops'`?

Reply:
Added `1.o` (zero vs the letter o)